lib/api/src/tests/stories.test.js

```diff
@@ -207,6 +207,25 @@ describe('stories API', () => {
       expect(navigate).toHaveBeenCalledWith('/story/a--1');
     });
 
+    it(
+      'navigates to the first child story of the parent story if a ' +
```

Comment:
@lonyele what's up with this string concat?

Reply:
It's not a prettier error(80 character) but this string became too long and I ended up concatenating like this. Is there  a better way? maybe template literal?

Reply:
I changed it to one line!